data/json/items/containers.json

```diff
@@ -128,6 +128,24 @@
     "color": "light_gray",
     "contains": 24,
     "flags": [ "TRADER_AVOID" ]
+	},
+	{
+    "id": "bag_zipper",
+    "type": "CONTAINER",
+    "category": "other",
+    "name": "zipper bag",
+    "name_plural": "zipper bags",
```

Comment:
This is redundant, as `s` will be added automatically in the end of `name` if no `name_plural` is set.

Reply:
Oh, I didn't notice that. Thanks for the review, I updated the file.